### PR TITLE
Add try/catch for two localStorage.getItem calls

### PR DESCRIPTION
**Pull request by ewjordan**
It seems that localStorage.getItem throws an exception in IE incognito mode, so this PR wraps a couple that have been showing up in New Relic with try/catch blocks, setting appropriate defaults when the calls fail.

Cleaning up our other usages and writing a safe wrapped function should be a longer term tech debt item (it would probably need to have the form getLocalStorage(key, valueIfLocalStorageUnavailable)), but it's not entirely trivial because defaults need to be set properly.

### CODE REVIEW

File: apps/src/lib/util/firehose.js [hunk 1]
```diff
@@ -115,14 +115,16 @@ class FirehoseClient {
...
+      }
+    } catch (e) {
+      // ignore, create ID below
     }
     let analytics_uuid = createUuid();
-    if (trySetLocalStorage("analyticsID", analytics_uuid)) {
-      return analytics_uuid;
-    }
-    return null;
+    trySetLocalStorage("analyticsID", analytics_uuid);
```
Comment (ashercodeorg) [inline apps/src/lib/util/firehose.js:122]: Is the format of the analytics UUID the same as the format of the local storage ID?

Comment (ewjordan) [reply apps/src/lib/util/firehose.js:122]: Yes, this is where that's populated.

Comment (ashercodeorg) [reply apps/src/lib/util/firehose.js:122]: Ah, right, never mind. Ducks in shame.

**Comment by ewjordan** [comment]
Worth noting: I did *not* verify this on IE, as I don't have a Windows box, but I did locally edit the code to check the code paths.

**Comment by ewjordan** [comment]
Re: Elijah's comment, no, no reason not to except that I hadn't figured out the right way to deal with the return value issue yet (for trySetLocalStorage you can just return true or false to know whether it succeeded; with tryGetLocalStorage I think the cleanest workaround is to require a default value to be passed, which you'll get back if the get fails).

I'll do this, it will make converting other usages easier, too.

**Comment by ewjordan** [comment]
PTAL, now with tryGetLocalStorage in utils

**Pull request merged**